.devcontainer/install-dependencies.sh

```diff
@@ -0,0 +1,25 @@
+python3.11 -m venv venv
+
+source venv/bin/activate
+
+pip install poetry
+
+poetry install --directory ./backend
+
+deactivate 
+
+echo "VITE_API_URL=https://bookish-space-parakeet-6q5jv596x56c5r5w-8001.app.github.dev">>./frontend/.env
```

Comment:
You mentioned this as an issue, I just a comment so we can mark as resolved once we know what to do